.changeset/dry-walls-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/primitives': minor
```

Comment:
Just curious how you determine the version. Why minor vs patch?

Reply:
@rezrah I figure we are adding a feature here, so minor is more appropriate. Let me know if you have some good guidelines as to when to choose which version.

Reply:
Yeah that makes sense. Thanks.